pages/1.13/security/oss/user-account-management/troubleshooting/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle: Troubleshooting
+title: Troubleshooting User Account Management
+excerpt: Troubleshooting user account management issues
+
+menuWeight: 50
+---
+
+# Debugging User Account Management
+
+## Admin Router
+
+Admin Router receives all user account management operations via HTTP and hands them over to the IAM. To debug user management problems, make sure Admin Router on the masters receives the HTTP request for the intended operation using the following commands.
+
+```bash
+sudo journalctl -u dcos-adminrouter.service
+```
+
+## Identity and Access Manager
+
+To debug user account management problems, check the IAM (Bouncer) on the masters using the following commands.
+
+```bash
+sudo journalctl -u dcos-bouncer.service
+```
+
+## CockroachDB
+
+The IAM stores user information in CockroachDB running on master nodes.
+If Admin Router and the IAM received the user management operation to perform according to their logs and the IAM could not fulfill the request in it likely that CockroachDB is in trouble. In this case check for errors or unusual patterns in the CockroachDB log on all master nodes.
+
+```bash
+sudo journalctl -u dcos-cockroach.service
+```
+
+The simplest way to determine whether a CockroachDB cluster is healthy is to query the CockroachDB node status information from a master node like this.
+
+```bash
+sudo /opt/mesosphere/bin/cockroach node status --ranges --host=$(/opt/mesosphere/bin/detect_ip) --insecure
+```
+
+The reported number of nodes should match the number of DC/OS master nodes in general. However each time a master node is replaced the CockroachDB node of the old DC/OS master node is not automatically removed.
+
+The number of `is_live` CockroachDB nodes should always match the current number of master nodes.
+
+DC/OS sets the number of replicas for CockroachDB ranges equal to the number of DC/OS master nodes. That means one master node should hold one copy of the data. Therefore, the number of `ranges_underreplicated` can go up temporarily if a master node goes out of business. After a new master node joins the cluster `ranges_underreplicated` is expected to go down again to zero after about 5 minutes.
```

Comment:
Rewrite `That means one master node should hold one copy of the data.` with `This implies  one master node should hold one copy of the data.

Reply:
Use `five` instead of 5 on `After a new master node joins the cluster `ranges_underreplicated` is expected to go down again to zero after about 5 minutes.`